### ISSUE TITLE
New release ?

### ISSUE THREAD

**Issue by tchapi** [area17/twill#480]
Hello !
Could we get a new release in a near future (1.2.3) ? It's been a few month and a few fixes have already been pushed, that it would be great to have — plus the 1.2.2 is now broken on PHP 7.4..

Thanks a lot !
Best

**Comment by philippdaun** [comment]
I agree it would be nice to have a steady release cycle for bug fixes. Currently, I usually require twill via specific commit hashes which feels a bit hacky but has become necessary.

**Comment by tchapi** [comment]
In fact I'm wondering if this project is still maintained ... ? Or does the original team seek maintainers ?

**Comment by brammittendorff** [comment]
Yes I'm wondering the same. https://github.com/area17/twill/issues/376

**Comment by ifox** [comment]
Hi everyone,

We're a little bit behind on wrapping up the release to tag it and announce it, but I wouldn't jump to the conclusion that Twill is not maintained anymore. Laravel 6 support has been available on master since mid-october, and development is very active on the project, both internally and externally. Please give `dev-master` a try and stay tuned, there's actually a lot coming if you haven't checked out what's been ongoing for the past 6 months:

- Laravel 6 support,
- Automated tests: Twill's PHP code is now more than 50% covered (starting from the bottom here, we had no tests just a few months ago)
- No webpack build necessary anymore, see https://github.com/area17/twill/commit/d88ab7a0a05e50019d51e5d9398866826eaa1b21,
- Simpler ways to store form fields, with automated browser and repeaters fields, and support for multiple fields in a single JSON column,
- A much smarter CLI,
- Azure upload support,
- Tons of bug fixes and developer experience improvements,
- And more that's going to be merged before the release, like i18n support (translating the CMS itself) or OAuth login support.

**Comment by karl007** [comment]
@ifox Sorry, but there seems a long way to a release.. the latest commit is months ago, there are many open pullRequests and some of these are required to get the `master` run (only tried to show the dashboard - NOTHING else) and some more to get rid of the Exceptions (see my PullRequest https://github.com/area17/twill/pull/506).

Everything tried on my system with `laravel 6.11`, and laravel 7 is on the edge.

So after an whole evening of trying to get twill run with laravel 6, I have to discourage from using twill with laravel6..  and also from hoping of a stable release very soon.. too many essential bugs :(

Maybe an option for the next project.

**Comment by philippdaun** [comment]
@karl007 While I agree that the release cycle of Twill has become a bit inscrutable (and is currently preventing me from recommending it for new projects) — your linked pull request doesn't even have a description. As a project maintainer, I wouldn't want to have to guess what a PR does exactly, no matter how simple.

@ifox I appreciate you maintaining this package while simultaneously juggling client work but I think you'd have less people on your backs about new releases if you stopped announcing them for "the next few days" if it's not going to happen. Most here probably prefer realistic expectation management — if you're not seeing a release coming anytime soon for lack of time, that's fine, but it'd be nice to know that so we can manage our clients' expectations in turn.

**Comment by karl007** [comment]
@philippdaun thank you, proper words, fully agree.. sorry for be a bit rough.. This is a OS project, so I thank the devs for giving their time.

What description would you expect? O only add missing namespaces at places where they are required (blade macros)

**Comment by tchapi** [comment]
Any news, by any chance ?

**Comment by arkid** [comment]
@ifox Laravel 5.8 security support has now ended since last [Wednesday the 26th of Feb](https://laravel.com/docs/6.x/releases#support-policy).   So Laravel 5.8 is now officially past it's EOL.    This is really quite serious I feel as it means naturally that Twill isn't out of the box able to support any 'current' Laravel version.   

This is going to really prohibit new people taking on Twill if this continues which would be a real shame and start reducing its uptake.

**Comment by tchapi** [comment]
Well, well ... I closed this when you released 2.0.0 and the promise of a release at least every month was quite appealing but to be honest I'm kind of disappointed now.

The community around this tool tries (_at least from my point of view_) to make quality issues and PRs (_I crafted a few myself, they're small but I hope they still help, and I've seen a lot of activity from @arkid and @philippdaun for instance_) but only few have been merged and there are still regressions from the previous versions that are not addressed (see https://github.com/area17/twill/issues/605#issuecomment-617224103 for instance)

I understand that we are living through strange times and that it can be challenging (for all of us developers, actually), but I think a few minor releases, even if they do not fix all the bugs, could give us all some air for our production projects.

I've said it before as well, but I think some contributors (_myself included_) would be open to help you maintain Twill more actively if ever you want some help.

Best
